opal/mca/common/ofi/common_ofi.c

```diff
@@ -356,7 +356,10 @@ static uint32_t get_package_rank(int32_t num_local_peers, uint16_t my_local_rank
                                        &pname, &locality_string, OPAL_STRING);
         if (OPAL_SUCCESS != rc || NULL == locality_string) {
             // If we don't have information about locality, fall back to procid
-            opal_show_help("help-common-ofi.txt", "package_rank failed", true);
```

Comment:
If you're deleting the use of this error message, then since this is the only place that this error message is displayed, it should be deleted out of `help-common-ofi.txt`.  And since it's the only message in `help-common-ofi.txt`, that file should probably be removed.  😄 

That being said, one of the benefits of `opal_show_help()` is that it is de-duplicated across multiple MPI processes when used with `mpirun` -- `opal_output[_verbose]()` is not de-duplicated.

Do you think it would be better to check the verbosity level manually and call `opal_show_help()` if the verbose level is high enough?  (and maybe put an annotation in the help message indicating that that message is only displayed at a given debug level, yadda yadda yadda -- because this is not the norm for show_help kinds of messages)

Reply:
Agreed. This was a vanilla cherry-pick from master; but I'll fix this in master first and update this PR.